EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+    The *Set text value* pop-up window is displayed.
+
+6. Enter the free text, for example "Please note our changed bank details". You can enter up to five lines.
+
+   ![Free text](../../Assets/Screenshots/EDI/Operation/ExportDefS_FTX.png "[Free text]")
+
+7. Click the [SAVE] button.   
+    The message structure is displayed.
+ 
+8. Specify the language used in the *D_3453 (Code specifying the language name)* row. To do this, click the ![Set constant value](../../Assets/Icons/Edit04.png "[Set constant value]") (Set constant value) button to the right of the *D_3453* row and specify the language code in the *Set text value* window. 
+
+9. Click the [SAVE] button.
+    - To the right of the rows that contain data, only the ![Delete attribute](../../Assets/Icons/Trash10.png "[Delete attribute]")(Delete attribute) button is available, with which you can remove the data, if required. The field name and description are no longer displayed in italics.  
+     - To the right of the segments that contain data, the ![Add condition](../../Assets/Icons/AddCondition.png) (Add condition) button is displayed with which you can define a condition for the output of this segment in the data mapping later.
+    - The segments that contain data are now highlighted with a small black border. This indicates that the segment contains data. 
+    
+10. If desired, add further *S_FTX* segments by clicking the ![Add XML attribute](../../Assets/Icons/Plus08.png "[Add XML attribute]") (Add XML attribute) button.   
+    An additional *S_FTX* segment is added below the currently processed segment.
+
+    ![Additional S_FTX segment](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefFTXAdditionalSegment.png "[Additional S_FTX segment]")
+
+
+## Specify conditions
+
+For each segment that contain data, you can define conditions under which the segment data is to be transferred to the message receiver. For example, you want to output a message free text only in case that a caution notice is to be added to the INVOIC. 
+
+This procedure shows how to specify a condition for the output of a message segment.
+
+#### Prerequisites
+
+- You have expanded the message structure, see [Get header file segments](#get-header-file-segments).
+- You have specified an S_FTX segment, see [Specify message free texts](#specify-message-free-texts).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Select message definition > File settings tab > Expand message structure*
+
+![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefDatesDangerousGoods.png "[Message structure]")
+
+1. Click the ![Add condition](../../Assets/Icons/AddCondition.png) (Add condition) button at the segment that you want to transfer only in case that a certain situation occurs.     
+    - To the right of the *S_FTX* segment, the ![Remove condition](../../Assets/Icons/Cross02.png) (Remove condition) button is displayed. You can use it to delete the condition, if desired.
```

Comment:
You can use it to delete the condition, if desired. -> Ist es "deleted" oder nur "removed", etwa deaktiviert? Wenn man wieder auf X klickt, wird wieder die Add condition-Symbol angezeigt, wie ein Ein-/Ausschalten.

Reply:
Steht in der nächsten Zeile, wo ich condition definieren kann